web/blog/2022-11-29-typescript-feature-announcement.md

```diff
@@ -33,9 +33,9 @@ Without further ado, let's see how we can use TypeScript with Wasp.
 Let's start by creating a fresh Wasp project:
 
 ```bash
-wasp new ts-project
+wasp new myApp
```

Comment:
Made this change since I felt like the name "ts-project" somehow implies that you need to put "ts" in name in order for it to work. Just a small nitpick 🤷